boards/riscv/tl321x/doc/index.rst

```diff
@@ -0,0 +1,224 @@
+.. _tl3218x:
+
+Telink TL3218X
+#####################
+
+Overview
+********
+
+The TL3218X Generic Starter Kit is a hardware platform which
+can be used to verify the `Telink TLSR9 series chipset`_ and develop applications
+for several 2.4 GHz air interface standards including Bluetooth 5.2 (Basic data
+rate, Enhanced data rate, LE, Indoor positioning and BLE Mesh),
+Zigbee 3.0, Homekit, 6LoWPAN, Thread and 2.4 Ghz proprietary.
+
+.. figure:: img/tl3218x.jpg
+     :align: center
+     :alt: TL3218X
+
+More information about the board can be found at the `Telink B92 Generic Starter Kit Hardware Guide`_ website.
```

Comment:
same here and in some more files old copy pasted names of board/SoC

Reply:
Changes to these contents have been planned. Will update them a bit later.